apps/frontend/src/pages/auth/libs/components/sign-up-form/sign-up-form.tsx

```diff
@@ -24,13 +24,15 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	const [isConfirmPasswordVisible, setIsConfirmPasswordVisible] =
 		useState<boolean>(false);
 
+	const navigate = useNavigate();
 	const handleFormSubmit = useCallback(
 		(event_: React.BaseSyntheticEvent): void => {
 			void handleSubmit((payload: UserSignUpFormDto) => {
 				const { confirmPassword, ...userData } = payload;
 
 				if (confirmPassword === userData.password) {
 					onSubmit(userData);
+					navigate("/");
```

Comment:
I think it's better if we use the useSelector hook to check if the user is authenticated.
`if (user) {
			navigate(AppRoute.ROOT);
}`